nupic/frameworks/opf/clamodel.py

```diff
@@ -703,18 +699,13 @@ def _handleCLAClassifierMultiStep(self, patternNZ,
 
     # Get the classifier input encoder, if we don't have it already
     if self._classifierInputEncoder is None:
-      if predictedFieldName is None:
-        raise RuntimeError("This experiment description is missing "
-              "the 'predictedField' in its config, which is required "
-              "for multi-step prediction inference.")
-
       encoderList = sensor.getSelf().encoder.getEncoderList()
       self._numFields = len(encoderList)
 
       # This is getting index of predicted field if being fed to CLA.
       fieldNames = sensor.getSelf().encoder.getScalarNames()
-      if predictedFieldName in fieldNames:
-        self._predictedFieldIdx = fieldNames.index(predictedFieldName)
+      if self._predictedFieldName in fieldNames:
+        self._predictedFieldIdx = fieldNames.index(self._predictedFieldName)
       else:
         # Predicted field was not fed into the network, only to the classifier
```

Comment:
does it make sense to run in this option, or should we raise an exception? As https://github.com/numenta/nupic/pull/2263/files#diff-5bf13635ee2cded8c4bfe689ea9a297dR728 fails and that encoder is used in most of the function here...